commons/src/main/java/org/infinispan/commons/marshall/Ids.java

```diff
@@ -8,7 +8,7 @@
  */
 public interface Ids {
    int MURMURHASH_2 = 71;
-   int MURMURHASH_2_COMPAT = 72;
+   // 72 used to be MURMURHASH_2_COMPAT
```

Comment:
I'm removing the id constant because AFAIK that's what we've always done when we've removed a class/externalizer...

But MurmurHash2 wasn't deprecated, so we can't remove it yet (nor its id).